concent_api/core/tests/test_utils.py

```diff
@@ -215,3 +217,27 @@ def test_that_method_returns_correct_format_name(output_format, expected):
     upper_output_format = adjust_format_name(output_format)
 
     assert_that(upper_output_format).is_equal_to(expected)
+
+
+class BlocksHelperTest(TestCase):
+    def setUp(self):
+        self.sci = mock.Mock()
+        self.blocks_helper = BlocksHelper(self.sci)
+
+    def test_get_latest_existing_block_at(self):
+        blocks = []
+        for i in range(9):
```

Comment:
this is a part of  test's setup as well: please either move it to `setUp` method or get rid of that method and set everything up here.

Reply:
Done. I just moved it to `setUp`